packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 52.8.0 - August 4, 2021
```

Comment:
Are we releasing today or tomorrow?

Reply:
@jag-sfdc It ALL depends on whether @AnanyaJha finishes the QA.

Reply:
oh no! QA is done, gus is not updated :) updated now!